ImperatorToCK3Tests/MapperTests/SuccessionLawMapper/SuccessionLawMapperTests.cpp

```diff
@@ -0,0 +1,41 @@
+#include "Mappers/SuccessionLawMapper/SuccessionLawMapper.h"
+#include "gtest/gtest.h"
+#include <sstream>
+
+
+
+TEST(Mappers_SuccessionLawMapperTests, nonMatchGivesEmptySet)
+{
+	std::stringstream input;
+	input << "link = { ck3 = ck3law imp = implaw }";
```

Comment:
Elsewhere you have the imp law first. Best to do the same here and in the other tests.